### ISSUE TITLE
Will this work with SimRacingStudio?

### ISSUE THREAD

**Issue by ObstacleTAKUYA** [tronicgr/AMC-open-hardware-servo-controller-4DOF#1]

**Comment by tronicgr** [comment]
No, this demo open hardware controller is missing the proper handshaking communication for SRS to detect and configure the controller for motion like it does for the AMC-AASD15A controller.

**Comment by ObstacleTAKUYA** [comment]
Ah i see. Would it be hard to implement from my side?
